stripe/src/main/java/com/stripe/android/model/Token.java

```diff
@@ -63,6 +64,25 @@ public Token(
         mBankAccount = bankAccount;
     }
 
+    /**
+     * Constructor that should not be invoked in your code.  This is used by Stripe to
+     * create tokens using a Stripe API response.
+     */
+    public Token(
+            String id,
+            boolean livemode,
+            Date created,
+            Boolean used
+    ) {
+        mId = id;
+        mType = TYPE_PII;
```

Comment:
It's not -- using this constructor means you're making a PII token (these aren't supposed to be used by actual users; but Java lacks a friend class and this needed to be built in another package)